### ISSUE TITLE
NTR: Creatine low in urine (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8282]
New term request Creatine low in urine (segal_110901180359):

**Comment by pnrobinson** [comment]
@MickeySegal
is this a 24h measurement? I do not think that a spot measurement would be as useful, but 24h measurement is used in the diagnosis of frailty? What is the context?

**Comment by MickeySegal** [comment]
https://www.ncbi.nlm.nih.gov/books/NBK3794/
- GAMT deficiency. Elevated guanidinoacetate (GAA) levels in urine, plasma, or cerebrospinal fluid (CSF) and low or low-normal creatine and creatinine levels in urine, plasma, or CSF
- AGAT deficiency. Low GAA levels in urine, plasma, or CSF and low or low-normal creatine and creatinine levels in urine, plasma, or CSF

**Comment by pnrobinson** [comment]
@MickeySegal Adding new term: 
- ID: HP:6000748
- label: Decreased urinary creatine level
- parents: [Abnormal urinary organic compound level (HP:6000531)](https://hpo.jax.org/app/browse/term/HP:6000531)
- pmid: 
- def: The amount of creatine in the urine, normalized for urine concentration, is below the lower limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8282
- synonyms:
